### DISCUSSION TITLE
Time-Based One Time Password 2fa support.

### DISCUSSION THREAD

**Discussion by Alien10123** [appwrite/appwrite#5681] [Ideas]
I think that TOTP (eg. google auth, etc) support should be added. For me, and probably some others, it is a deal-breaker that only phone 2fa is supported. I think that TOTP, or email 2fa should be added. As far as I know, there is no way to do this in functions, but if there is could you let me know? I have looked at #907, but that is more focused on other things.

Thanks

**Comment by joeyouss** [comment]
Hi, probably in plans.
I think you might want to see this PR to know the status: https://github.com/appwrite/rfc/pull/56

**Reply by Alien10123** [reply]
I didn't see that before. It seems to be what I'm looking for, I just need to wait for it to be merged. Thanks!

**Comment by tripolskypetr** [comment]
![image](https://github.com/appwrite/appwrite/assets/19227776/f75eec6a-aec2-47d4-aeba-512449754277)

Hey! Nice docs! Who need static markdown in the github repo! For a great product, we only need nice first look on our server-side rendered landing page

**Reply by tripolskypetr** [reply]
The best when It deployed automatically without manual check with CI/CD. The test coverage effectively solves everything
